ratbag/drivers/sinowealth.py

```diff
@@ -72,8 +73,8 @@ def from_bytes(cls, data: bytes) -> "Config":
             Spec("BBBBBB", "?"),
             Spec("B", "config"),
             Spec("B", "dpi_count"),  # two nibbles!
-            Spec("B", "dpi_enabled"),
             Spec("B", "dpi", repeat=16),
+            Spec("B", "dpi_disabled_slots"),  # bit mask
```

Comment:
this doesn't match the `sinowealth_config_report` where we have dpi_count, then the dpi. Note that the `Spec` entries are parsed in-order, so one of this or libratbag seems off now.

the `Spec` approach doesn't allow for nibbles so config and dpi_count here both parse one byte that then gets split into report_rate and active_dpi as well.

Reply:
Yes, thank you. I renamed a bunch of fields but did not stage it, so things I did stage got a bit out of order.